pangram/pangram.py

```diff
@@ -1,2 +1,9 @@
 def is_pangram(sentence):
-    pass
+    
+    # ensure string is lowercase
+    sentence = sentence.lower()
+    
+    # remove some punctuation
+    for thing in ' ,.!?:;()':
+        sentence = sentence.replace(thing,'')
+    return set(sentence) == set("abcdefghijklmnopqrstuvwxyz")
```

Comment:
Maybe use some string similarity measure such as the number of character matches over the number of characters in the alphabet? 

In that way you do not need to worry about punctuation and numbers. If all alphabet letters appear at least once, they will match the alphabet and the result of the division will be one.

Reply:
The most recent code passes all unit tests.